crates/astria-core/src/sequencer/v1/mod.rs

```diff
@@ -43,6 +43,7 @@ pub const ADDRESS_LEN: usize = 20;
 pub const ROLLUP_ID_LEN: usize = 32;
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize))]
 pub struct Address([u8; ADDRESS_LEN]);
```

Comment:
Do we not want `#[cfg_attr(feature = "serde", serde(serialize_with = "crate::serde::base64"))]` here too?